src/main/scala/com/microsoft/hyperspace/index/sources/default/DefaultFileBasedSource.scala

```diff
@@ -253,7 +268,7 @@ class DefaultFileBasedSource(private val spark: SparkSession) extends FileBasedS
   override def hasParquetAsSourceFormat(logicalRelation: LogicalRelation): Option[Boolean] = {
     logicalRelation.relation match {
       case HadoopFsRelation(_: PartitioningAwareFileIndex, _, _, _, format, _)
-        if isSupportedFileFormat(format) =>
+          if isSupportedFileFormat(format) =>
```

Comment:
nit: Is indentation here correct?

Reply:
yes it's fixed by scalafmt; it's because this `if` is for the `case`.